kernel/src/table_changes/scan.rs

```diff
@@ -267,14 +267,14 @@ mod tests {
             scan.all_fields,
             vec![
                 ColumnType::Selected("id".to_string()),
-                ColumnType::Selected("_commit_version".to_string()),
+                ColumnType::Selected(COMMIT_VERSION_COL_NAME.to_string()),
```

Comment:
but this one could (should) change so we can test the actual `_commit_version` col name